package.json

```diff
@@ -8,7 +8,7 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "test:watch": "jest --watchAll",
-    "test": "TZ=GMT jest",
+    "test": "SET TZ=GMT && jest --runInBand --forceExit",
```

Comment:
Why changing it?

Reply:
![image](https://user-images.githubusercontent.com/52848645/128151467-a2d13c52-40ee-4839-a7db-514c8f8497b0.png)
because I got this error when using the original `TZ=GMT jest`

Reply:
Hum that's annoying, on my side I've got this error if I run `SET TZ=GMT && jest`
![Screenshot 2021-08-04 at 10 58 33](https://user-images.githubusercontent.com/8709134/128153162-09041754-68af-45a1-9756-cd59b6864795.png)
I've opened an [issue](https://github.com/NMF-earth/nmf-app/issues/294). In the mean time, you can develop locally with `"test": "SET TZ=GMT && jest --runInBand --forceExit",` and then when you are done just revert to `"test": "TZ=GMT jest",` and add `--no-verify` when commiting

Reply:
Thanks, I will make all the requested changes above and recreate pull request again.